### DISCUSSION TITLE
Multi-field variogram estimation

### DISCUSSION THREAD

**Discussion by maximepollet27** [GeoStat-Framework/GSTools#320] [Q&A]
Hi all,

I have been trying out GSTools for a particular application (it's great btw! :)), and I have a question regarding multi-field variogram estimation. I have 9 different fields that should therefore have similar properties. The issue is that the point sets are different for each field, which means that I can't use the technique described in the "Multi-field variogram estimation" section on the website. What would be a good approach to estimate the variogram of these fields? Would it make sense to fit a variogram for each field, and then average them?
My final goal is to generate random fields from the covariance model thus obtained in case that helps.

Thanks in advance!

**Comment by LSchueler** [comment]
Hi maximepollet27,

I'm sorry for the late reply.
Did you already have a look at the different variograms when estimating them for the individual fields? Maybe you could try averaging the estimated parameters to come up with a variogram for all the fields?
